### PR TITLE
spec: Fix dom4j definition

### PR DESCRIPTION
**Pull request by arachmani**
fixes - https://github.com/oVirt/ovirt-dwh/issues/36

Signed-off-by: Asaf Rachmani <arachman@redhat.com>

### CODE REVIEW

**Comment by avlitman** [comment]
Hi @arachmani Thanks for the fix.
We removed fedora dependencies for ovirt-dwh in this commit:https://github.com/oVirt/ovirt-dwh/commit/2a058d7feb010a2bb02c2daad80539d06972629d. 

So not sure if we want to fix, I will consult Martin.

File: ovirt-engine-dwh.spec.in [hunk 1]
```diff
@@ -20,14 +20,20 @@
...
-        %global dom4j ovirt-engine-wildfly
+        %define rhv_build 0
     %else
-        %global dom4j eap7-dom4j
+        %define rhv_build 1
     %endif
 %endif
 
+%global dom4j ovirt-engine-wildfly
+%if 0%{rhv_build}
```
Comment (avlitman) [inline ovirt-engine-dwh.spec.in:33]: Not sure if we need the '0' here

Comment (mwperina) [reply ovirt-engine-dwh.spec.in:33]: `0` is not needed as default value is provided on line 23

File: ovirt-engine-dwh.spec.in [hunk 2]
```diff
@@ -20,14 +20,20 @@
...
+%define rhv_build 0
 %if 0%{?rhel}
     %if 0%{?centos}
-        %global dom4j ovirt-engine-wildfly
+        %define rhv_build 0
     %else
-        %global dom4j eap7-dom4j
+        %define rhv_build 1
     %endif
 %endif
```
Comment (mwperina) [inline ovirt-engine-dwh.spec.in:30]: Maybe we could make it even more clear:

```
%if 0%{?rhel}
%if 0%{?centos}
# Build package for oVirt on CentOS Stream
%define rhv_build 0
%else
# Build package for RHV on RHEL
%define rhv_build 1
%endif
%else
# Default to oVirt build for all other platforms
%define rhv_build 0
%endif
```

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**